vars/main.yml

```diff
@@ -39,6 +39,10 @@ jenkins_plugins_dir: "{{ jenkins_home }}/plugins"
 # Directory for secret files (credentials, etc)
 jenkins_secrets_dir: "{{ jenkins_root }}/secrets"
 
+# Name of the Jenkins service, as listed by Ansible's service facts. This is only used
+# when inspecting the output of the host's services.
+jenkins_service_name: "jenkins.service"
```

Comment:
I still don't think we should have a var that is specific to only one service manager, then override it in `tasks/stop-service.yml`.

Reply:
It's not specific to a service manager. As the comment notes, it's specific to the way that Ansible lists services (which is coincidentally the same way that systemd refers to the service). The comment (correctly, IMHO) doesn't mention a specific service manager for that reason.

The root of the problem is that when we are creating the service, that name must be `jenkins` for SysV init. For systemd, I suppose it could be either `jenkins` or `jenkins.service`, but we use the former. However, when we gather service facts, Ansible refers to the name as `jenkins.service`, regardless of whether SysV init or systemd is being used (I suspect that this some systemd bias in Ansible itself, but that's beside the point). That's why we need this variable to help us find the correct service name after gathering service facts.

Reply:
Let's get rid of the var to reduce confusion, since we aren't using it when starting or configuring the service. It's a bit clunky, but see a15320632eee4a450e839f76d111e8aa1ea9e000.